src/eventsub/conduit/shard/disabled.rs

```diff
@@ -0,0 +1,98 @@
+#![doc(alias = "conduit.shard.disabled")]
+//! A conduit shard is disabled by twitch.
+
+use super::*;
+
+/// [`conduit.shard.disabled`](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#conduitsharddisabled)
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "typed-builder", derive(typed_builder::TypedBuilder))]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct ConduitShardDisabledV1 {
+    /// Your application’s client id. The provided client_id must match the client ID in the application access token.
+    #[cfg_attr(feature = "typed-builder", builder(setter(into)))]
+    pub client_id: String,
+    /// Optional. The conduit ID to receive events for. If omitted, events for all of this client’s conduits are sent.
+    #[cfg_attr(feature = "typed-builder", builder(setter(into)))]
+    pub conduit_id: Option<String>,
```

Comment:
should probably make this a typed string with `twitch_types` out of scope for this pr though